course-preparation/Ansible/group-vars/agondway.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: Walu Gondwe                          # Real name of the person
+    org: MUST/Malawi                       # Organisation/team they work in
```

Comment:
This '/' might create problems for  Ansible. Can you please just update the var to "MUST Malawi" or either of  them (MUST or Malawi).